Kconfig.zephyr

```diff
@@ -486,6 +486,14 @@ config BUILD_OUTPUT_UF2_FAMILY_ID
 	  name string. If the SoC in use is known by UF2, the Family ID will
 	  be pre-filled with the known value.
 
+config BUILD_OUTPUT_UF2_USE_FLASH_BASE
+	bool
+	default y
```

Comment:
I think this and `BUILD_OUTPUT_UF2_USE_FLASH_OFFSET` should default to false, as it doesn't make sense to have both enabled. Also, for the rpi, the defconfig looks like this:
```
config BUILD_OUTPUT_UF2_USE_FLASH_BASE
	default y if RP2_REQUIRES_SECOND_STAGE_BOOT
	default n

config BUILD_OUTPUT_UF2_USE_FLASH_OFFSET
	default y if !RP2_REQUIRES_SECOND_STAGE_BOOT
	default n
```
The `default n` are required, because if the condition is false, it fallbacks to the `default y`